### DISCUSSION TITLE
How to search a particular library using `item.search` JSON-RPC?

### DISCUSSION THREAD

**Discussion by mhucka** [retorquere/zotero-better-bibtex#1955] [Q&A] [answered]
The documentation indicates that one of the endpoints is [`item.search`](https://retorque.re/zotero-better-bibtex/exporting/json-rpc/#itemsearchsearchterms). Is it possible to search a specific library using this? I can't figure out the syntax to do so and can't find any examples of how to do it.

**Comment by retorquere** [comment, marked as answer]
This is not currently possible, but is easy to add. If you open an issue for this, I can get you a build to test.
